ner_v2/tests/numeral/number/en/number_ner_tests.yaml

```diff
@@ -0,0 +1,131 @@
+tests:
+  en:
+#  - _id_: en_1
```

Comment:
Why are some of these commented?

Reply:
They were written for a function other than `NumberDetector`.
I will do all other functions later.
This is just for detectors.